timed/reports/views.py

```diff
@@ -268,11 +268,16 @@ def _create_workreport(self, from_date, to_date, today, project, reports, user):
 
     def list(self, request, *args, **kwargs):
         queryset = self.filter_queryset(self.get_queryset())
+
+        if queryset.count() == 0:
+            return HttpResponseBadRequest()
```

Comment:
I'm not sure that this "bad request" is warranted. In my opinion, an empty list is a valid output and shouldn't cause an error

Reply:
Since this endpoint is used for exports only it might make sense to prompt the user that nothing has been selected.
Doesn't make much sense to export an empty excel sheet.

Reply:
Anyway I made this a restful response which is actually helpful for the frontend.